src/qz/printer/info/WindowsPrinterMap.java

```diff
@@ -12,7 +12,7 @@
 public class WindowsPrinterMap extends NativePrinterMap {
     private static final Logger log = LogManager.getLogger(WindowsPrinterMap.class);
 
-    public synchronized NativePrinterMap putAll(PrintService[] services) {
+    public synchronized NativePrinterMap putAll(PrintService... services) {
```

Comment:
Why is this changed to vararg?

Reply:
Oh, vararg allows arrays and direct args, neat. Ok, disregard.

Reply:
so PrintServiceMatcher can put just the default printer if nothing else has been searched for yet without having to create an array of one

Reply:
> so PrintServiceMatcher can put just the default printer if nothing else has been searched for yet without having to create an array of one

Thanks.  Yeah, after staring at it for a while, I realized it was a shortcut to allow one value or an array of values, which is a shortcut I never thought of using. :)